terraform/shared/container_networking/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "cf_password" {
+  type      = string
+  sensitive = true
+}
+variable "cf_user" {}
+variable "cf_org_name" {}
+variable "cf_space_name" {}
+variable "source_app_name" {}
+variable "destination_app_name" {}
+variable "destination_port" {
+  type    = string
+  default = "61443"
```

Comment:
61443 is the port for cloud.gov provided TLS termination for internally routed traffic.

Reply:
If you want, you can toss in a comment linking to https://cloud.gov/docs/management/container-to-container/